azure-pipelines.yml

```diff
@@ -86,3 +98,26 @@ jobs:
           assetUploadMode: 'replace'
           addChangeLog: false
           isPrerelease: true
+      - bash: |
+          set -eux pipefail
+          echo "##vso[task.setvariable variable=status;isOutput=true]$(Agent.JobStatus)"
+        name: status
+
+  - job: notify
+    dependsOn: ["linux", "macos", "windows", "release"]
+    condition: always() #eq(variables['Build.SourceBranchName'], 'master')
```

Comment:
I guess you only disabled this for testing?

Reply:
Yes, will definitely re-enable before merging.